db/DB.js

```diff
@@ -296,6 +296,15 @@ const DB = {
     return updatedSubscriber;
   },
 
+  async updateFxAProfileData(subscriber, fxaProfileData) {
+    await knex("subscribers").where("id", subscriber.id)
+    .update({
+      fxa_profile_json: fxaProfileData,
+    })
+    .returning("*");
+    return this.getSubscriberById(subscriber.id);
```

Comment:
Q: On the previous line we use `.returning("*")`, but then ignore the [presumably] returned row/record. Now here we're doing _another_ query to get a subscriber by `subscriber.id`. Is that needed, or can we just return the result of `await knex().update()`?

http://knexjs.org/#Builder-returning

Reply:
We can probably drop the `returning("*")` call. But I'm calling `this.getSubscriberById` because it calls `joinEmailAddressesToSubscriber` which is required for the `req.user` and `req.session.user` objects.